pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -481,6 +483,11 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 		}
 	}
 
+	err := purgeOpenStackLbPoolMember(client, poolId, addresses)
```

Comment:
the variable err already exists, no need to recreate it, just to update.

Reply:
ack.